spec/oas3.patched/components/schemas/Device.yaml.rej

```diff
@@ -0,0 +1,11 @@
+--- spec/oas3.patched/components/schemas/Device.yaml
++++ spec/oas3.patched/components/schemas/Device.yaml
+@@ -119,6 +119,8 @@ properties:
```

Comment:
This patch failed to apply and should probably be upstreamed.

Reply:
The failed patch was updated in place to unblock this PR.  It has also been upstreamed, so we will be able to remove this patch when the upstream change is released to production.